src/lib/paths.ts

```diff
@@ -120,17 +122,51 @@ export const ensureFluenceAquaServicesDir = async (): Promise<string> =>
 export const ensureFluenceAquaDeployedAppPath = async (): Promise<string> =>
   path.join(await ensureFluenceAquaDir(), DEPLOYED_APP_AQUA_FILE_NAME);
 
-export const ensureFluenceJSAppPath = (fluenceJSDir: string): string =>
-  path.join(fluenceJSDir, APP_JS_FILE_NAME);
+// JS
 
-export const ensureFluenceJSDeployedAppPath = (fluenceJSDir: string): string =>
-  path.join(fluenceJSDir, DEPLOYED_APP_JS_FILE_NAME);
+export const ensureFluenceJSAppPath = async (
```

Comment:
Are these paths configurable? I mean JS and TS destination paths.

Reply:
@DieMyst WDYT?

@folex  WDYT about recommended naming notation?

Reply:
now yes

Reply:
awesome